README.md

```diff
@@ -66,6 +66,20 @@ When done making the changes on the document,
 
 Once the PR is reviewed and merged, the changes will appear on Graph Explorer in 2 working days.
 
+### Please note that:
+1. The order of properties is as outlined below <br>
```

Comment:
You can add the text in brackets...
`The order of properties is as outlined below (where applicable for your sample):`